atlas/AtlasMl/tests/test_artemis_client.py

```
@@ -0,0 +1,125 @@
+
+        with patch('httpx.Client') as mock_client:
+            mock_client.return_value.__aenter__.return_value.get.return_value = mock_response
+
+            result = await artemis_client.health_check()
+            assert result is True
+
+    @pytest.mark.asyncio
+    async def test_health_check_failure(self, artemis_client):
+        """Test failed health check."""
+        with patch('httpx.Client') as mock_client:
+            mock_client.return_value.__aenter__.return_value.get.side_effect = Exception("Connection failed")
```

Ruff (0.12.2)

Line too long (109 > 88)

(E501)